internal/web3ext/web3ext.go

```diff
@@ -821,26 +821,6 @@ web3._extend({
 			name: 'addBalance',
 			call: 'les_addBalance',
 			params: 2
-		}),
-		new web3._extend.Method({
```

Comment:
Shouldn't we keep these and maybe return an error?
2 things here
1) it will be weird for the user, maybe the user won't understand if it is an error or not if an rpc call disapeared 
2) technically, this should be effective from the gFork, not before. So everyone that updated to this version, should be able to use this until the activation

Maybe it's better to keep this, with a if-fork between the an error from the gFork, and the actual behaviour and prune it after the checkpoint...

Reply:
You're right. I took the easy route assuming that we only had few transactions using that path, but I fixed it to do the proper hardfork check.